src/logger.cr

```diff
@@ -58,11 +58,12 @@ class Logger
     UNKNOWN
   end
 
-  alias Formatter = String, Time, String, String, IO ->
+  alias Formatter = Severity, Time, String, String, IO ->
 
   private DEFAULT_FORMATTER = Formatter.new do |severity, datetime, progname, message, io|
-    io << severity[0] << ", [" << datetime << " #" << Process.pid << "] "
-    io << severity.rjust(5) << " -- " << progname << ": " << message
+    label = severity.unknown? ? "ANY" : severity.to_s
```

Comment:
I just copy-pasted this line from previous version. Why not just `severity.to_s` without branching into other symbols?